.schema/config.schema.json

```diff
@@ -138,6 +138,15 @@
             "8eaef023-2b34-4da1-9baa-8bc8c9d6a490",
             "contoso.onmicrosoft.com"
           ]
+        },
+        "requested_claims": {
+          "title": "OpenID Connect claims",
+          "description": "The OpenID Connect claims and optionally their properties which should be included in the id_token or returned from the UserInfo Endpoint.",
+          "type": "string",
```

Comment:
I would like to have this as an object and validate it further. That ensures at least it is valid JSON.

Reply:
Done